src/model_signing/_signing/sign_certificate.py

```diff
@@ -116,8 +122,17 @@ def __init__(
 
         self._store = crypto.X509Store()
         for certificate in certificates:
+            if self.fingerprints:
+                self.display_cert_fingerprint(certificate, hashes.SHA256())
             self._store.add_cert(crypto.X509.from_cryptography(certificate))
 
+    def display_cert_fingerprint(self, cert, hash_alg: hashes.Hash) -> None:
+        fp = cert.fingerprint(hash_alg)
+        print(
```

Comment:
I'm really debating if we should use print or just logging. This is in the library, and we'd like to test it at some point.

Would it be too much work to switch to logging instead?